Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang
```

Comment:
Would it make sense to make this Dockerfile more similar to the ones maintained by the Prometheus project, e.g. https://github.com/prometheus/node_exporter/blob/master/Dockerfile ?